src/coreComponents/physicsSolvers/wavePropagation/AcousticWaveEquationSEM.cpp

```diff
@@ -146,6 +145,11 @@ void AcousticWaveEquationSEM::postProcessInput()
                   "Invalid number of physical coordinates for the receivers",
                   InputError );
 
+  EventManager const & event = this->getGroupByPath< EventManager >( "/Problem/Events" );
+  real64 const & maxTime = event.getReference< real64 >( EventManager::viewKeyStruct::maxTimeString() );
```

Comment:
Not that this should be fixed in this PR, but we should really not be accessing things like this by traversing the data repository.

Reply:
Yes we've discussed this with François, but since this is done in multiple places over GEOSX, we decided to put it this way for now